src/Hangfire.PostgreSql/PostgreSqlBootstrapperConfigurationExtensions.cs

```diff
@@ -22,9 +23,25 @@ public static class PostgreSqlBootstrapperConfigurationExtensions
         }
 
         /// <summary>
-        /// Tells the bootstrapper to use PostgreSQL as a job storage
-        /// with the given options, that can be accessed using the specified
-        /// connection string.
+        /// Tells the bootstrapper to use PostgreSQL as a job storage that can be accessed
+        /// using a connection from the given connection builder.
+        /// </summary>
+        /// <param name="configuration">Configuration</param>
+        /// <param name="connectionBuilder">Connection builder</param>
+        [PublicAPI]
+        public static PostgreSqlStorage UsePostgreSqlStorage(
```

Comment:
You can also add `UsePostgreSqlStorage(this GlobalConfiguration configuration, Action<NpgsqlConnection> connectionSetup)` overload here and wrap delegate by `DefaultConnectionBuilder`.

Reply:
Good idea, I'll add that later